e2e_tests/tests/fixtures/custom_searcher/model_noop.py

```diff
@@ -0,0 +1,259 @@
+import collections
+import json
+import logging
+import os
+import pathlib
+import pickle
+import random
+import sys
+import time
+from typing import Any, Dict, Optional, Type
+
+import numpy as np
+
+import determined as det
+from determined import layers, tensorboard, util, workload
+from determined.common import check
```

Comment:
Do not use `check` module in any new code.

It's a terrible antipattern; it raises a meaningless error type, it doesn't work with mypy, and even though it's specifically intended to always deliver usable error messages, about half of our usages of it result in actually useless error messages, like:

```python
check.true(os.path.isfile("startup-hook-ran"), "File should exists.")
```

Which will print `CheckFailed: expected true but got false, file should exist`... What file?

Reply:
Yes please!

Reply:
(I haven't read that file in years... I didn't know it said that)